docs/user-guide/limitrange/README.md

```diff
@@ -131,7 +132,8 @@ $ kubectl get pods --namespace=limit-example
 POD           IP           CONTAINER(S)   IMAGE(S)   HOST          LABELS      STATUS    CREATED          MESSAGE
 nginx-ykj4j   10.246.1.3                             10.245.1.3/   run=nginx   Running   About a minute
                            nginx          nginx                                Running   54 seconds
-$ kubectl get pods nginx-ykj4j --namespace=limit-example -o yaml | grep resources -C 5
+$ kubectl get pods nginx-ykj4j --namespace=limit-example -o yaml |\
```

Comment:
Pipe should be on newline according to our style guide:

https://google-styleguide.googlecode.com/svn/trunk/shell.xml?showone=Pipelines#Pipelines